cypress-shared/integration/location-availability/2.6-Verify-location-availability props.spec.js

```diff
@@ -0,0 +1,57 @@
+import { UsDetails1 } from '../../support/shopper-location/outputvalidation'
+import {
+  updateRetry,
+  preserveCookie,
+  loginAsAdmin,
+  loginAsUser,
+} from '../../support/common/support'
+import locationAvailabilityProducts from '../../support/location-availability/product'
+import selectors from '../../support/common/selectors'
+
+const { country1, postalCode1 } = UsDetails1
+
+describe('Verify-Location-availability', () => {
+  // Load test setup
+  before(() => {
+    loginAsAdmin()
+    cy.getVtexItems().then((vtex) => {
+      loginAsUser(vtex.robotMail, vtex.robotPassword)
+    })
+  })
+
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip(
+    'Adding Location & Verify location Availability',
+    updateRetry(1),
+    () => {
+      cy.addNewLocation(country1, postalCode1)
+      cy.get('div[class*=vtex-modal-layout]').should('not.be.visible')
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(10000)
```

Comment:
Do we really need to wait 10s ?
Can u pls check is it possible to replace this with intercept